testproject/Assets/Tests/Runtime/MultiprocessRuntime/TestCoordinator.cs

```diff
@@ -54,6 +57,48 @@ public void Start()
     {
         MultiprocessLogger.Log("Start");
         bool isClient = Environment.GetCommandLineArgs().Any(value => value == MultiprocessOrchestration.IsWorkerArg);
+        string[] args = Environment.GetCommandLineArgs();
```

Comment:
This is not a PR blocker, but would it make sense to make the MaxWaitTimeoutSec and PerTestTimeoutSec something that could be configured by the command line since you are already parsing the args?